lib/OpenQA/Utils.pm

```diff
@@ -666,7 +668,7 @@ sub check_download_whitelist {
         @okdomains = split(/ /, $whitelist);
     }
     for my $param (keys %$params) {
-        next unless ($param =~ /_URL$/);
+        next unless ($param =~ ASSET_URL_REGEX);
```

Comment:
I'd suggest to make use of `asset_type_from_setting` - if it returns '' on the stripped param, it's not an asset.

Reply:
Seems like we partially already have that, see https://github.com/os-autoinst/openQA/pull/2469/files#diff-a8857daf563deea1776f5c009cd73a32L724

Reply:
So? close the PR?

Reply:
But we go out early in https://github.com/os-autoinst/openQA/pull/2469/files#diff-a8857daf563deea1776f5c009cd73a32L712 . I guess we can simply change that line from "log_warning" to "log_debug" as I consider it not a problem that any openQA admin in the background would need to care about.

Reply:
not close but change to a trivial change "warn" -> "debug"